frontend/packages/knative-plugin/src/utils/__mocks__/traffic-splitting-utils-mock.ts

```diff
@@ -1,87 +1,60 @@
-import { RevisionModel, ServiceModel } from '../../models';
+import * as _ from 'lodash';
+import {
+  MockKnativeResources,
+  revisionObj,
+} from '@console/dev-console/src/components/topology/__tests__/topology-knative-test-data';
 import { RevisionKind, ServiceKind as knativeServiceKind } from '../../types';
 
-export const mockTrafficData = [
-  { percent: 50, tag: 'tag-1', revisionName: 'rev-1' },
-  { percent: 50, tag: 'tag-2', revisionName: 'rev-2' },
+export const mockServiceData: knativeServiceKind = _.cloneDeep(
+  MockKnativeResources.ksservices.data[0],
+);
+
+export const mockUpdateTrafficData = [
+  { percent: 25, tag: 'tag-1', revisionName: 'overlayimage-fdqsf' },
+  { percent: 25, tag: 'tag-2', revisionName: 'overlayimage-tkvz5' },
+  { percent: 25, tag: 'tag-3', revisionName: 'overlayimage-bwpxq' },
+  { percent: 25, tag: 'tag-4', revisionName: 'overlayimage-n2b7n' },
 ];
-export const mockServiceData: knativeServiceKind = {
-  apiVersion: `${ServiceModel.apiGroup}/${ServiceModel.apiVersion}`,
-  kind: ServiceModel.kind,
-  spec: {
-    traffic: [
-      {
-        percent: 50,
-        revisionName: 'rev-1',
-        tag: 'tag-1',
-      },
-      {
-        percent: 50,
-        revisionName: 'rev-2',
-        tag: 'tag-2',
-      },
-    ],
-  },
-  status: {
-    traffic: [
-      {
-        latestRevision: false,
-        percent: 100,
-        revisionName: 'rev-1',
-        tag: 'tag-1',
-      },
-    ],
-  },
-};
 
-export const mockUpdateRequestObj: knativeServiceKind = {
-  apiVersion: `${ServiceModel.apiGroup}/${ServiceModel.apiVersion}`,
-  kind: ServiceModel.kind,
-  spec: {
-    traffic: [
-      {
-        percent: 50,
-        revisionName: 'rev-1',
-        tag: 'tag-1',
-      },
-      {
-        percent: 50,
-        revisionName: 'rev-2',
-        tag: 'tag-2',
-      },
-    ],
+export const mockTraffic = [
+  {
+    percent: 25,
+    revisionName: 'overlayimage-fdqsf',
+    tag: 'tag-1',
```

Comment:
why `mockTraffic` , above you already have  `mockUpdateTrafficData`

Reply:
refactored the mock file